src/DrupalCacheAdapter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Capgemini\Cache;
+
+/**
+ * @class DrupalCacheAdapter
+ * @package Capgemini\Cache
+ * @codeCoverageIgnore
```

Comment:
This trait explicitly calls core's cache functions, which will require a bootstrapped Drupal instance.